### ISSUE TITLE
Add JDBC reader&writer

### ISSUE THREAD

**Issue by 2efPer** [ritchie46/polars#473]
Is it possible to add JDBC reader and writer?

**Comment by ritchie46** [comment]
Do you specifically want a Java db connector?

**Comment by alippai** [comment]
While I don't understand the original request about JDBC, I think a python ODBC example in docs using TurboODBC would make sense. I'd assume that's the fastest way now to retrieve or insert data to a DB from polars.

**Comment by ritchie46** [comment]
Yes, I've watched TurboODBC as well. A colleague tried it, but had compiling errors. The fact that it doesn't have wheels made it less than ideal for python. 

I hope to eventually start a seperate crate that can indeed create Arrow columnar memory in a performant manner.

**Comment by 2efPer** [comment]
Sorry for my fat finger, I mean a database connection reader and writer. I didn't find a way to read or write to the database easily.

**Comment by ritchie46** [comment]
For Python we will try to figure this out for TurboODBC and then include it in the docs. You could already try it yourself if you find time. The idea is to create a pyarrow table with TurboODBC and then use that to convert to polars.
